### PR TITLE
fix(StatusStickersPopup): fix stick pack list not scrollable

### LINKED ISSUE #8989: Sticker button is not fully displayed in the sticker popup
# Bug Report

## Description

The sticker button is not fully displayed in the sticker popup

Screenshot 2023-01-06 at 17 39 45

## Steps to reproduce

1. Click add sticker button (near emoji and gif buttons)
2. Download the sticker if you don't have it
3. Send any sticker
4. Open the sticker popup again

#### Expected behavior

Sticker icon displaying correctly

#### Actual behavior

Sticker icon corrupted

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #8989

image

image

### CODE REVIEW

File: ui/imports/shared/status/StatusStickersPopup.qml [hunk 1]
```diff
@@ -256,10 +256,12 @@ Popup {
             StatusScrollView {
                 id: installedStickersSV
                 height: 40
+                Layout.fillWidth: true
+                ScrollBar.vertical.policy: ScrollBar.AlwaysOff
+                clip: true
```
Comment (caybro) [inline ui/imports/shared/status/StatusStickersPopup.qml:261]: We do clip the StatusScrollView by default already :)
```qml
// NOTE: in Qt6 clip true will be default
clip: true
```

**Review by caybro — APPROVED** [review]
LGTM

**Review by 0x-r4bbit — APPROVED** [review]

**Review by mprakhov — APPROVED** [review]

File: ui/imports/shared/status/StatusStickersPopup.qml [hunk 2]
```diff
@@ -256,10 +256,11 @@ Popup {
             StatusScrollView {
                 id: installedStickersSV
```
Comment (alexandraB99) [inline ui/imports/shared/status/StatusStickersPopup.qml:257]: I guess this id is not needed anymore, could you please remove?

File: ui/imports/shared/status/StatusStickersPopup.qml [hunk 3]
```diff
@@ -256,10 +256,11 @@ Popup {
...
                 height: 40
+                Layout.fillWidth: true
+                ScrollBar.vertical.policy: ScrollBar.AlwaysOff
 
                 RowLayout {
                     id: stickersRowLayout
-                    width: installedStickersSV.availableWidth
                     spacing: Style.current.padding
```
Comment (alexandraB99) [inline ui/imports/shared/status/StatusStickersPopup.qml:264]: could you please check with a `Tracer` that the `RowLayout` here has its dimensions set properly?

**Pull request merged**